server/controllers/feedbackController.js

```diff
@@ -0,0 +1,26 @@
+const feedBackModel = require('../models/Feedback');
+
+function createFeedback(req, res) {
+  const feedback = feedBackModel.addFeedback(req);
+  feedback.then((data) => {
```

Comment:
Our style is inconsistent across the current code - the team should sync on if we want to use .then or async/await, and then make it consistent